src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4242,10 +4251,22 @@ public override bool PreVisit(CallNode node)
                 nodeInput.Accept(this);
 
                 // At this point we know the type of the first argument, so we can check for untyped objects
-                if (overloadWithUntypedObjectLambda != null && _txb.GetType(nodeInput) == DType.UntypedObject)
+                if (_txb.GetType(nodeInput) == DType.UntypedObject)
                 {
-                    maybeFunc = overloadWithUntypedObjectLambda;
-                    scopeInfo = maybeFunc.ScopeInfo;
+                    if (overloadWithUntypedObjectLambda != null)
+                    {
+                        maybeFunc = overloadWithUntypedObjectLambda;
+                        scopeInfo = maybeFunc.ScopeInfo;
+                    }
+                    else
+                    {
+                        UntypedObjectScopeError(node, maybeFunc, nodeInput);
```

Comment:
Comment about why we don't allow 1st param UO+lambda without a specific overload?